src/Raven.Studio/wwwroot/App/views/database/indexes/editIndex.html

```diff
@@ -12,6 +12,9 @@
                         <a data-bind="click: getCSharpCode, visible: isEditingExistingIndex" class="btn" title="Copy the index c# code"><i class="icon-code"></i><span>Copy C#</span></a>
                         <a data-bind="attr: { href: queryUrl }, visible: isEditingExistingIndex" class="btn" title="Query the index"><i class="icon-query"></i><span>Query</span></a>
                         <a data-bind="attr: { href: termsUrl }, visible: isEditingExistingIndex" class="btn" title="Navigate to index terms"><i class="icon-terms"></i><span>Terms</span></a>
+                        <!-- TODO replace the arrow-down icon when RavenDB-17594 is done-->
+                        <!-- TODO add access level for DatabaseAdmin on v5.2 -->
```

Comment:
please put that TODO's inside issue and/or create new to avoid being set as fixed, as we miss that

Reply:
created new issue:
https://issues.hibernatingrhinos.com/issue/RavenDB-17595